### ISSUE TITLE
/connections/create-invitation params mismatch the ACA-Py API

### ISSUE THREAD

**Issue by haidinhtuan** [hyperledger-labs/acapy-java-client#88]
acapy-java-client version: 0.7.33
acapy version: 0.7.5

I’m currently using the library in a Spring Boot project and have encountered an issue with the `CreateInvitationParams` class when trying to use it for creating invitation.

I’m trying to set the `isPublic` field in `CreateInvitationParams`, expecting it to serialize to a URL parameter named `public`. However, it serializes to `is_public` instead, which the API I’m calling does not recognize.

Here’s a simple code snippet showing how I’m using it:

```
CreateInvitationRequest request = CreateInvitationRequest.builder()
                    .serviceEndpoint("http://1.2.3.4:8101")
                    .myLabel("AccessPoint")
                    // .mediationId(null)
                    // .metadata(null)
                    // .recipientKeys(null)
                    // .routingKeys(null)
                    .build();

            CreateInvitationParams params = CreateInvitationParams.builder()
                    .alias(null) // or provide a value
                    .autoAccept(false)
                    .multiUse(false)
                    .isPublic(true)
                    .build();

            Optional<CreateInvitationResponse> response = ac.connectionsCreateInvitation(request, params);
```
            
 Is there a recommended way to handle this situation, or could this be considered for a future update? Any guidance or temporary solutions would be greatly appreciated.

Thank you in advance!

**Comment by etschelp** [comment]
This was a issue with the builders serialisation. I just released a new version of the library 0.8.0 that contains a fix for that. The new version contains the changes to work with aca-py 0.8.0, but should also work with the aca-py 0.7.5 version. If I have the time I will l start working on making it compatible with aca-py 0.10.x

**Comment by haidinhtuan** [comment]
Thanks a lot @etschelp. That was lightning fast!
